contributors/working_with_datastructures.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: documentation
+title:  Working with Datastructures
+order: 2
+---
+
+## 
+
+Check out the example on how to work with a table [here](https://github.com/Caleydo/lineage/blob/master/src/dataExplorations.ts). 
```

Comment:
I don't like this approach linking to a repository which code changes. I would prefer a clean hello world repository without anything else